src/features/user/Widget/DonationLink/DonationLinkForm.tsx

```diff
@@ -197,10 +197,10 @@ const DonationLinkForm = ({
                 onChange={() => {
                   setIsSupport(!isSupport);
                 }}
-                disabled={user.isPrivate}
+                disabled={user?.isPrivate}
               />
             </InlineFormDisplayGroup>
-            {user.isPrivate && (
+            {user?.isPrivate && (
```

Comment:
Since this is within the dashboard, `user` would already exist. If not, the component should ideally not render. 

Perhaps it makes sense to keep the check on the user right before the whole render statement with `isArrayUpdated && ready`?